pkgs/by-name/vo/voicevox-engine/package.nix

```diff
@@ -0,0 +1,106 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
```

Comment:
Because of splicing dark magic, use `python3Packages` instead of `python3.pkgs`.

If you need the python interpreter, use `python3Packages.python`.